target/linux/realtek/files-6.12/drivers/net/ethernet/rtl838x_eth.c

```diff
@@ -1519,9 +1519,6 @@ static int rtl931x_chip_init(struct rtl838x_eth_priv *priv)
 	/* Enable ESD auto recovery */
 	sw_w32(0x1, RTL931X_MDX_CTRL_RSVD);
 
-	/* Init SPI, is this for thermal control or what? */
```

Comment:
Remove RTL931X_SPI_CTRL0 from mach include too.

Reply:
Ok, done. Please check the change